src/modules/function/ret.rs

```diff
@@ -56,12 +57,15 @@ impl SyntaxModule<ParserMetadata> for Return {
 
 impl TranslateModule for Return {
     fn translate(&self, meta: &mut TranslateMetadata) -> String {
-        let (name, id, variant) = meta.fun_name.clone().expect("Function name not set");
+        let fun_name = meta.fun_meta.as_ref()
+            .map(FunctionMetadata::mangled_name)
+            .expect("Function name not set");
```

Comment:
Missed a rename & error message

Reply:
Re "missed a rename", I deliberately kept the local variable as `fun_name` because it's the (mangled) function name.

Re "error message", I kept the error message the same because I was trying to make minimal changes.

Reply:
Ok, changed to "Function name and return type not set". I'm not changing the local variable name though!